tools/generate_png_images.js

```diff
@@ -6,28 +6,38 @@
   `tools/visual_regression.sh`.
 */
 const { JSDOM } = require('jsdom');
-const dom = new JSDOM(`<!DOCTYPE html></html>`);
-window = dom.window;
-document = dom.window.document;
-
 const fs = require('fs');
-const [scriptDir, imageDir] = process.argv.slice(2, 4);
 
-const Vex = require(`${ scriptDir }/vexflow-debug.js`);
-Vex.Flow.Test = require(`${ scriptDir }/vexflow-tests.js`);
+fs.readFile('build/webComponents-debug.js', 'utf8', function(err, data) { 
```

Comment:
Let's rename `data` to `webComponentsSource` or something like that, so that when we `window.eval(webComponentsSource)` will be more readable.